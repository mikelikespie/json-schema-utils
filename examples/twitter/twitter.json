[
  {
    "coordinates": null,
    "favorited": false,
    "created_at": "Tue Jul 13 17:38:21 +0000 2010",
    "truncated": false,
    "entities": {
      "urls": [
        {
          "expanded_url": null,
          "url": "http://bit.ly/94BCpx",
          "indices": [
            119,
            139
          ]
        }
      ],
      "hashtags": [
 
      ],
      "user_mentions": [
        {
          "name": "Matt Harris",
          "id": 777925,
          "indices": [
            56,
            70
          ],
          "screen_name": "themattharris"
        },
        {
          "name": "Abraham Williams",
          "id": 9436992,
          "indices": [
            72,
            80
          ],
          "screen_name": "abraham"
        },
        {
          "name": "Matt Kelly",
          "id": 6300552,
          "indices": [
            82,
            93
          ],
          "screen_name": "mattwkelly"
        }
      ]
    },
    "text": "Just opened more slots for SF @socialappworkshop - with @themattharris, @abraham, @mattwkelly from FB- going fast tho! http://bit.ly/94BCpx",
    "contributors": null,
    "annotations": null,
    "id": 18454047411,
    "geo": null,
    "in_reply_to_user_id": null,
    "place": null,
    "in_reply_to_screen_name": null,
    "user": {
      "name": "Shanley Kane",
      "profile_sidebar_border_color": "BDDCAD",
      "profile_background_tile": false,
      "profile_sidebar_fill_color": "DDFFCC",
      "created_at": "Wed Jun 24 22:24:43 +0000 2009",
      "profile_image_url": "http://a0.twimg.com/profile_images/887285836/lele2_normal.jpg",
      "location": "San Francisco",
      "profile_link_color": "0084B4",
      "follow_request_sent": false,
      "url": "http://www.shanleykane.wordpress.com",
      "favourites_count": 8,
      "contributors_enabled": false,
      "utc_offset": -28800,
      "id": 50462250,
      "profile_use_background_image": false,
      "profile_text_color": "333333",
      "protected": false,
      "followers_count": 479,
      "lang": "en",
      "notifications": false,
      "time_zone": "Pacific Time (US & Canada)",
      "verified": false,
      "profile_background_color": "9AE4E8",
      "geo_enabled": true,
      "description": "i work at apigee. this makes me an apigeek. i tweet about cloud, agile, APIs, tech stuff, pandas, propaganda, snark and trying to make it in the Valley.",
      "friends_count": 440,
      "statuses_count": 3100,
      "profile_background_image_url": "http://s.twimg.com/a/1279228556/images/themes/theme1/bg.png",
      "following": false,
      "screen_name": "ShanleyKane"
    },
    "source": "<a href=\"http://www.tweetdeck.com\" rel=\"nofollow\">TweetDeck</a>",
    "in_reply_to_status_id": null
  },
  {
    "coordinates": null,
    "favorited": false,
    "created_at": "Sun Jul 11 02:45:34 +0000 2010",
    "truncated": false,
    "entities": {
      "urls": [
 
      ],
      "hashtags": [
 
      ],
      "user_mentions": [
        {
          "name": "Matt Harris",
          "id": 777925,
          "indices": [
            0,
            14
          ],
          "screen_name": "themattharris"
        },
        {
          "name": "Twitter Espa\u00f1ol",
          "id": 85426644,
          "indices": [
            15,
            26
          ],
          "screen_name": "twitter_es"
        },
        {
          "name": "Paulina Goto",
          "id": 110086132,
          "indices": [
            27,
            42
          ],
          "screen_name": "soypaulinagoto"
        }
      ]
    },
    "text": "@themattharris @twitter_es @soypaulinagoto holaaaa!",
    "contributors": null,
    "annotations": null,
    "id": 18240214048,
    "geo": null,
    "in_reply_to_user_id": 777925,
    "place": null,
    "in_reply_to_screen_name": "themattharris",
    "user": {
      "name": "leonardo daniel ",
      "profile_sidebar_border_color": "87bc44",
      "profile_background_tile": false,
      "profile_sidebar_fill_color": "e0ff92",
      "created_at": "Sun Jul 11 02:35:10 +0000 2010",
      "profile_image_url": "http://s.twimg.com/a/1279228556/images/default_profile_5_normal.png",
      "location": null,
      "profile_link_color": "0000ff",
      "follow_request_sent": false,
      "url": null,
      "favourites_count": 0,
      "contributors_enabled": false,
      "utc_offset": null,
      "id": 165262200,
      "profile_use_background_image": true,
      "profile_text_color": "000000",
      "protected": false,
      "followers_count": 0,
      "lang": "es",
      "notifications": false,
      "time_zone": null,
      "verified": false,
      "profile_background_color": "9ae4e8",
      "geo_enabled": false,
      "description": null,
      "friends_count": 3,
      "statuses_count": 3,
      "profile_background_image_url": "http://s.twimg.com/a/1279228556/images/themes/theme1/bg.png",
      "following": false,
      "screen_name": "pulpin_000"
    },
    "source": "web",
    "in_reply_to_status_id": null
  },
  {
    "coordinates": null,
    "favorited": false,
    "created_at": "Tue Jul 06 19:49:45 +0000 2010",
    "truncated": false,
    "entities": {
      "urls": [
        {
          "expanded_url": null,
          "url": "http://wiki.github.com/themattharris/annotations-hackfest/",
          "indices": [
            0,
            58
          ]
        }
      ],
      "hashtags": [
        {
          "text": "devnest",
          "indices": [
            109,
            117
          ]
        }
      ],
      "user_mentions": [
        {
          "name": "Matt Harris",
          "id": 777925,
          "indices": [
            22,
            36
          ],
          "screen_name": "themattharris"
        }
      ]
    },
    "text": "http://wiki.github.com/themattharris/annotations-hackfest/ Twitter annotations wiki page from @themattharris #devnest",
    "contributors": null,
    "annotations": null,
    "id": 17891652167,
    "geo": null,
    "in_reply_to_user_id": null,
    "place": null,
    "in_reply_to_screen_name": null,
    "user": {
      "name": "Paul Kane",
      "profile_sidebar_border_color": "ffffff",
      "profile_background_tile": false,
      "profile_sidebar_fill_color": "ffffff",
      "created_at": "Thu May 03 05:58:32 +0000 2007",
      "profile_image_url": "http://a3.twimg.com/profile_images/98546479/PK-front_normal.jpg",
      "location": "London",
      "profile_link_color": "8b8b9c",
      "follow_request_sent": false,
      "url": "http://pk2004.posterous.com/",
      "favourites_count": 988,
      "contributors_enabled": false,
      "utc_offset": 0,
      "id": 5732282,
      "profile_use_background_image": true,
      "profile_text_color": "000000",
      "protected": false,
      "followers_count": 1516,
      "lang": "en",
      "notifications": false,
      "time_zone": "London",
      "verified": false,
      "profile_background_color": "FFFFFF",
      "geo_enabled": true,
      "description": "Tech geek, gadget addict, music lover, MTB'er, runner - sometimes.",
      "friends_count": 1212,
      "statuses_count": 3202,
      "profile_background_image_url": "http://a3.twimg.com/profile_background_images/8321573/banksy.jpg",
      "following": false,
      "screen_name": "pk2004"
    },
    "source": "web",
    "in_reply_to_status_id": null
  }
]
