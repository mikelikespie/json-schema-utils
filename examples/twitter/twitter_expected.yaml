properties:
  annotations: {type: 'null'}
  contributors: {type: 'null'}
  coordinates: {type: 'null'}
  created_at: {type: string}
  entities:
    properties:
      hashtags:
        items:
          properties:
            indices:
              items: {type: integer}
              type: array
            text: {type: string}
          type: object
        type: array
      urls:
        items:
          properties:
            expanded_url: {type: 'null'}
            indices:
              items: {type: integer}
              type: array
            url: {type: string}
          type: object
        type: array
      user_mentions:
        items:
          properties:
            id: {type: integer}
            indices:
              items: {type: integer}
              type: array
            name: {type: string}
            screen_name: {type: string}
          type: object
        type: array
    type: object
  favorited: {type: boolean}
  geo: {type: 'null'}
  id: {type: integer}
  in_reply_to_screen_name:
    type: [string, 'null']
  in_reply_to_status_id: {type: 'null'}
  in_reply_to_user_id:
    type: [integer, 'null']
  place: {type: 'null'}
  source: {type: string}
  text: {type: string}
  truncated: {type: boolean}
  user:
    properties:
      contributors_enabled: {type: boolean}
      created_at: {type: string}
      description:
        type: [string, 'null']
      favourites_count: {type: integer}
      follow_request_sent: {type: boolean}
      followers_count: {type: integer}
      following: {type: boolean}
      friends_count: {type: integer}
      geo_enabled: {type: boolean}
      id: {type: integer}
      lang: {type: string}
      location:
        type: [string, 'null']
      name: {type: string}
      notifications: {type: boolean}
      profile_background_color: {type: string}
      profile_background_image_url: {type: string}
      profile_background_tile: {type: boolean}
      profile_image_url: {type: string}
      profile_link_color: {type: string}
      profile_sidebar_border_color: {type: string}
      profile_sidebar_fill_color: {type: string}
      profile_text_color: {type: string}
      profile_use_background_image: {type: boolean}
      protected: {type: boolean}
      screen_name: {type: string}
      statuses_count: {type: integer}
      time_zone:
        type: [string, 'null']
      url:
        type: [string, 'null']
      utc_offset:
        type: [integer, 'null']
      verified: {type: boolean}
    type: object
type: object
